test/functional/apps/discover/_shared_links.js

```diff
@@ -20,6 +20,11 @@ bdd.describe('shared links', function describeIndexTests() {
 
   bdd.before(function () {
     baseUrl = PageObjects.common.getHostPort();
```

Comment:
instead of using the getHostPort method and replacing the ports, adding another method called getHost() that returns without ports.  my thoughts are mostly around naming, it's slightly confusing to use a method with get ports and then remove them